site/realm/functions/setCreatedAtField.js

```diff
@@ -0,0 +1,7 @@
+exports = async function ({ changeEvent, serviceName, dbName }) {
```

Comment:
I think we can remove the `serviceName` param since it will be the same for each collection in our project. It's safe to hardcode it here with the value: `mongodb-atlas`